tutorials/02-functions/01-calling-functions.md

```diff
@@ -54,7 +54,9 @@ var logSomethingExpression = function (something) {
 };
 ```
 
-We'll revisit some of the other differences between function declarations and expressions later on when we discuss [recursion](#Recursion).
+We'll revisit some of the other differences between function declarations and expressions later on when we discuss [recursion][].
+
+[recursion]: https://github.com/indiesquidge/advanced-js-fundamentals-ck/blob/md-fixes/tutorials/02-functions/04-recursion.md#function-declarations-function-expressions-and-recursion
```

Comment:
Ah, I can't believe how many times I did this. All the links should be fixed now.